### ISSUE TITLE
(not an issue...help request only)

### ISSUE THREAD

**Issue by Jock90** [prb28/vscode-amiga-assembly#254]
(Apologies for posting here (I have tried other forums but not getting anywhere), will clean up/delete later) 

I'm looking to resurrect an old Amiga1200 platform game creator assembly program I wrote almost 30 years ago. It assembles (Asm-One) and runs ok on my real Amiga1200, and assembles and runs ok in WinUAE using PCMCIA card to transfer (there's a small demo game built in). In looking at getting my brain working better these days by playing about with/extending this program using a modern IDE (on the PC), I installed VS-Code and this fantastic Amiga Assembly program. Unfortunately, back in the day, I stupidly had Asm-One set on case insensitive, and have different case versions for my labels (variables, addresses, branches/jumps etc) and am now met with with the following...

`Linker error: Link Error 21(PlatformDesigner+0x1de8): Reference to undefined symbol setcondDelay`

Here, setcondDelay also appears as SETcondDelay. Standardising one label shows more and more issues, and there's a lot of labels with over 7000 lines of code. 
To get this running quickly, is there a case insensitive switch? I'm struggling to see one in VS-Code (it might be FS-UAE that's the issue).

Ultimately, I'd like to find the label table Asm-One is using, then write a program to scan the source and standardise them, though it would be nice if there was an existing option.

Again, sorry for posting here.

I'm ok with dropboxing the source.

**Comment by torkildl** [comment]
These is one in vasm, the assembler used: -nocaseSee vasm manual for more info.Sendt fra min Galaxy
-------- Opprinnelig melding --------Fra: Jock90 ***@***.***> Dato: 04.03.2023  08:54  (GMT+01:00) Til: prb28/vscode-amiga-assembly ***@***.***> Ko: Subscribed ***@***.***> Emne: [prb28/vscode-amiga-assembly] (not an issue...help request only) (Issue #254) 
(Apologies for posting here (I have tried other forums but not getting anywhere), will clean up/delete later)
I'm looking to resurrect an old Amiga1200 platform game creator assembly program I wrote almost 30 years ago. It assembles (Asm-One) and runs ok on my real Amiga1200, and assembles and runs ok in WinUAE using PCMCIA card to transfer (there's a small demo game built in). In looking at extending this program using a modern IDE (on the PC), I installed VS-Code and this fantastic Amiga Assembly program. Unfortunately, back in the day, I stupidly had Asm-One set on case insensitive, and have different case versions for my labels (variables, addresses, branches/jumps etc) and am now met with with the following...
Linker error: Link Error 21(PlatformDesigner+0x1de8): Reference to undefined symbol setcondDelay
Here, setcondDelay also appears as SETcondDelay. Standardising one label shows more and more issues, and there's a lot of labels with over 7000 lines of code.
To get this running quickly, is there a case insensitive switch? I'm struggling to see one in VS-Code (it might be FS-UAE that's the issue).
Ultimately, I'd like to find the label table Asm-One is using, then write a program to scan the source and standardise them, though it would be nice if there was an existing option.
Again, sorry for posting here.
